src/figuresClasses.ts

```diff
@@ -1,19 +1,106 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
Create separate types for `shape` and `color`. You'll need to use them then. Seems that you can even use enum here